commands/common/functions.py

```diff
@@ -0,0 +1,53 @@
+# Imported Modules
+from common.database.models import Users, Servers, ServerToUsers
+from sqlalchemy.orm import Session
+
+
+# USER ENTITY
+def returnUserEntity(discord_id: str, session: Session):
+    # queries user entity
+    user = session.query(Users).filter_by(discord_id=discord_id).first()
+    
+    if not user:
+        # entity doesn't exist; create new entity
+        user = Users(discord_id=discord_id)
+        session.add(user)
+        session.commit()
+        print(f"User ID ({discord_id}) added to the database")
+
+    # returns entity
+    print(f"Returned User ID ({discord_id}) from the database")
+    return user
+
+
+# SERVER ENITTY
+def returnServerEntity(server_id: str, session: Session):
+    # queries server entity
+    server = session.query(Servers).filter_by(server_id=str(server_id)).first()
+
+    if not server:
+        # entity doesn't exist; create new entity
+        server = Servers(server_id=str(server_id))
+        session.add(server)
+        session.commit()
+        print(f"Server ID ({server_id}) added to the database")
+
+    print(f"Returned Server ID ({server_id}) from the database")
+    return server
+
+
+# SERVERTOUSERS ENTITY
+def returnServerToUsersEntity(discord_id: str, server_id: str, session: Session):
```

Comment:
The datatypes.py allows you to instead (for function parameters) just need one thing, since its a class.

So instead of having discord_id: str and server_id: str
you can have
data: ServersToUsers
and you just use that to get your discord and server id.

This is just a comment, nothing has to be fixed here.

Reply:
Wouldn't that mean we need to create a datatype class and set each of its attributes before calling the function each time?

Reply:
Basically yes. It might be more helpful if some database calls requires more information that just the discord and server id. Instead of having 5 or 6 params in the function header.